puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -61,6 +61,17 @@
     package { 'createrepo':
       ensure => present,
     }
+
+    yumrepo { 'pulpcore-3.16':
```

Comment:
Should we scope this to EL7 or is it also needed on EL8?

Reply:
no, el8 has a sufficiently new version, so can be el7 only

Reply:
I just merged https://github.com/theforeman/foreman-infra/pull/1720 so let's try to keep it ready for EL8 :)